src/storage/src/render/upsert/types.rs

```diff
@@ -450,6 +458,7 @@ impl UpsertStateBackend for InMemoryHashMap {
             stats.processed_gets += 1;
             let value = self.state.get(&key).cloned();
             let size = value.as_ref().map(|v| v.memory_size());
+            stats.processed_gets_size += size.unwrap_or(0);
```

Comment:
can we also count the number that are non-none?

Reply:
Oh I was doing it for rocksdb, not for upsert level. Maybe I should make it upsert level instead